src/generators/minecraft-cat-character/MinecraftCatCharacterGenerator.res

```diff
@@ -51,8 +48,46 @@ let script = () => {
   let oi = 0
   let oo = 0
 
-  // Background
-  Generator.drawImage("Background", (0, 0))
+  // Overlay region variables
+  let hideHelmet = Generator.getBooleanInputValue("Hide Helmet")
+  let hideJacket = Generator.getBooleanInputValue("Hide Jacket")
+  let hideLeftSleeve = Generator.getBooleanInputValue("Hide Left Sleeve")
+  let hideRightSleeve = Generator.getBooleanInputValue("Hide Right Sleeve")
+  let hideLeftPant = Generator.getBooleanInputValue("Hide Left Pant")
+  let hideRightPant = Generator.getBooleanInputValue("Hide Right Pant")
+  let tailType =
+    Generator.getSelectInputValue("Tail Type")->Belt.Int.fromString->Belt.Option.getWithDefault(1)
+
+  let cycleTailTypes = t => {
+    let t = if t === 4 {
```

Comment:
Nice!

No need to change, but there is a function `mod()` that helps with this kind of cycling.

```res
let cycleTailTypes = t => {
  let t = mod(t, 4) + 1
  Belt.Int.toString(t)
}
```
Also, not sure if you know about the `defineText()` function yet - if you would like to show a message, such as:

```res
Generator.defineText(
  "Click the tail to change its style. Currently using tail type " ++ Belt.Int.toString(tailType),
)
```